AsyncDisplayKit/Details/ASCollectionViewLayoutController.mm

```diff
@@ -164,4 +165,19 @@ - (CGRect)rangeBoundsWithScrollDirection:(ASScrollDirection)scrollDirection
   return CGRectExpandToRangeWithScrollableDirections(rect, tuningParameters, _scrollableDirections, scrollDirection);
 }
 
+- (ASRangeTuningParameters)tuningParametersForRangeType:(ASLayoutRangeType)rangeType
+{
+    BOOL isVisible = ASInterfaceStateIncludesVisible(_collectionView.interfaceState);
+    BOOL isScrolling = (_collectionView.scrollDirection != ASScrollDirectionNone);
```

Comment:
How should we think about the distinction between a view that has appeared and never been scrolled before, versus a view that has been scrolled and come to a stop?

In the latter case, we probably don't want to restrict the Display range once it stops and destroy objects that were prepared in case the user scrolls again, because it is likely in that case. I wonder if there is room for two different variables here, Or an additional bit in the bit field